### PR TITLE
debug:edit time zone problem

### PR DESCRIPTION
**Pull request by shinyu0430**

### CODE REVIEW

File: utils/track.js [hunk 1]
```diff
@@ -94,11 +94,13 @@ module.exports = {
 
       return formattedData;
     }
-
+    
     async function getRecentSubmissions(channelId, users) {
-      const todayDate = DateTime.local().toFormat("yyyy-MM-dd");
+      const todayDate = moment().tz("Asia/Taipei").format("YYYY-MM-DD");
+      console.log(todayDate)
```
Comment (i-am-harveyt) [inline utils/track.js:100]: Don't forget to remove `console.log`

File: utils/track.js [hunk 2]
```diff
@@ -94,11 +94,13 @@ module.exports = {
...
       result = `# ${todayDate}\n`;
       for (const user of users) {
+        console.log(user)
```
Comment (i-am-harveyt) [inline utils/track.js:103]: This may need to be remove either

**Review by i-am-harveyt — APPROVED** [review]
It looks fine overall. But please remove the `console.log`s, thanks for your effort.

**Comment by i-am-harveyt** [comment]
Emm... So how's the `console.log`s going?

**Comment by shinyu0430** [comment]
I'll modify it later QQ. ( I want to make sure the bot can keep executing now)

**Pull request merged**